### ISSUE TITLE
A glitch in navigation 

### ISSUE THREAD

**Issue by parthi2929** [0xGG/crossnote-app#105]
I am trying this app for past few days as I am fan of your vscode preview extension. Wish I could use your extension for full documentation of multiple pages (like using latex for book/article/journal). Waiting for that day. 

I tried using this app in edit mode. When I click on a space below images, the navigation messes up, and goes somewhere down the place I clicked. Thinking cursor might be still on the place I clicked, I tried using arrow keys to confirm, but cursor as well was misplaced. I have shown in below gif. Can you please have a look and fix? I prefer split view but when you have images, both views do not line up properly, so trying to adapt this flow as well. 

![crossnote_nav_issue_01](https://user-images.githubusercontent.com/11896263/82070557-8cff1200-96f2-11ea-8c96-ff5d000ad936.gif)

**Comment by shd101wyy** [comment]
Yeah I also encountered that issue. I will take a closer look when I can. Thank you for reporting the problem

**Comment by dongshunyao** [comment]
The project is being refactored, so we will close outdated issues. Once the new version is released, please open a new issue if you encounter any problems. Thank you for your contributions and support!
